apps/parsley/src/utils/logRowTypes/index.ts

```diff
@@ -0,0 +1,26 @@
+import { ProcessedLogLine, SkippedLinesRow } from "types/logs";
+
+/**
+ * `isSkippedLinesRow` determines if a row is a SkippedLinesRow.
+ * @param logLine - the processed log line to check
+ * @returns true if the row is a SkippedLinesRow
+ */
+
+const isSkippedLinesRow = (
+  logLine: ProcessedLogLine,
+): logLine is SkippedLinesRow =>
+  typeof logLine === "object" &&
+  "rowType" in logLine &&
```

Comment:
you can short circuit this and remove the `"rowType" in logLine` assertion. If the logLine is not an object it will fail and the only other option is a `SkippedLinesRow` due to the typing of `ProcessedLogLine`
```suggestion
  typeof logLine === "object" &&
```